### DISCUSSION TITLE
Interactive Table Action Dropdown

### DISCUSSION THREAD

**Discussion by curateship** [codyhouse/codyhouse-framework#118] [Q&A] [answered]
The "Edit" link is selected after clicking on the dropdown. Is this a error? and how do I make it unselected?

![image](https://user-images.githubusercontent.com/66774849/151470962-9af67aec-697c-4d4a-8dea-1648be943742.png)

**Comment by claudia-romano** [comment, marked as answer]
Hi, this is a focus effect. When you open the dropdown, the focus is moved to the first item (to improve accessibility). 

The component uses the Menu component to create the dropdown; if you want to modify the focus effect, you can do that in the CSS of the Menu component:
https://codyhouse.co/ds/components/app/menu

I'm closing this discussion. If you have problems with the component, you can open an issue on the component folder:
https://github.com/CodyHouse/cd-interactive-table-changelog/issues

Thanks
